src/sentry/api/endpoints/project_templates_index.py

```diff
@@ -60,3 +63,37 @@ def get(self, request: Request, organization: Organization) -> Response:
             on_results=lambda x: serialize(x, request.user, ProjectTemplateSerializer()),
             paginator_cls=OffsetPaginator,
         )
+
+    @ensure_rollout_enabled(PROJECT_TEMPLATE_FEATURE_FLAG)
+    def post(self, request: Request, organization: Organization) -> Response:
+        """
+        Create a new Project Template.
+
+        Create a new project template for the organization.
+        """
+        name = request.data.get("name", None)
```

Comment:
I'm guessing project template will be have more body parameters eventually, would it be useful to move `name` and `options` into a serializer to setup for that?

Reply:
fair, i was being lazy 🤣

Reply:
was able to find a simpler example and updated. thanks for the push 🎉